CHANGELOG.md

```diff
@@ -14,6 +14,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
+* Add new parameter to specify minimum TLS version [#1103](https://github.com/elastic/package-registry/pull/1103)
```

Comment:
Should we add another entry for the error that is raised now when setting the values from Environment values ?

Reply:
Could be added, yes, as you prefer.

Reply:
Added a new bugfix entry for this using the same PR link... maybe should it be as an enhancement? @jsoriano

Reply:
Bugfix is fine.